src/main/java/dev/openfga/sdk/api/StreamedListObjectsApi.java

```diff
@@ -0,0 +1,75 @@
+package dev.openfga.sdk.api;
+
+import static dev.openfga.sdk.util.Validation.assertParamExists;
+
+import dev.openfga.sdk.api.client.ApiClient;
+import dev.openfga.sdk.api.client.ApiResponse;
+import dev.openfga.sdk.api.client.HttpRequestAttempt;
+import dev.openfga.sdk.api.client.StreamingResponseBody;
+import dev.openfga.sdk.api.configuration.Configuration;
+import dev.openfga.sdk.api.model.ListObjectsRequest;
+import dev.openfga.sdk.errors.ApiException;
+import dev.openfga.sdk.errors.FgaInvalidParameterException;
+import dev.openfga.sdk.telemetry.Attribute;
+import dev.openfga.sdk.telemetry.Attributes;
+import java.net.http.HttpRequest;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * API handler for streamed list objects operations.
+ * This class is separate from the generated OpenFgaApi to avoid modifications to generated code.
+ */
+public class StreamedListObjectsApi {
+    private final Configuration configuration;
```

Comment:
`configuration` is assigned but never read. Should it be merged with `requestConfiguration` when making request, or handled like other requests?